proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/queryable/ExportMetaDataExecutor.java

```diff
@@ -110,6 +116,20 @@ private String generateRulesData(final Collection<RuleConfiguration> rules) {
         return result.toString();
     }
     
+    private void generateSnapshotInfo(final ShardingSphereMetaData metaData, final ExportedClusterInfo exportedClusterInfo) {
+        Optional<GlobalClockRule> globalClockRule = metaData.getGlobalRuleMetaData().findSingleRule(GlobalClockRule.class);
```

Comment:
Please use `getSingleRule` to get global rule, because it always exists.

Reply:
Fixed, THX